### PR TITLE
CAT-1904 Added description field expansion in program overview

### PR DESCRIPTION
**Pull request by bernadettespieler**

### CODE REVIEW

**Comment by bernadettespieler** [comment]
ok to test

File: catroid/res/layout/dialog_text_multiline_dialog.xml [hunk 1]
```diff
@@ -21,30 +21,36 @@
   ~ You should have received a copy of the GNU Affero General Public License
   ~ along with this program.  If not, see <http://www.gnu.org/licenses/>.
   -->
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
     style="@style/DefaultDialog"
-    android:orientation="vertical" >
+    android:paddingBottom="0dp"
```
Comment (thomasmauerhofer) [inline catroid/res/layout/dialog_text_multiline_dialog.xml]: 0dp? Is this line necessary?

File: catroid/res/layout/dialog_text_multiline_dialog.xml [hunk 2]
```diff
@@ -21,30 +21,36 @@
...
+        style="@style/DefaultDialog"
+        android:orientation="vertical"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
 
-    <View style="@style/DefaultDialog.Line" />
+        <org.catrobat.catroid.ui.CapitalizedTextView
+            style="@style/DefaultDialog.TextView"
+            android:layout_width="wrap_content"
+            android:layout_marginRight="0dp"
```
Comment (thomasmauerhofer) [inline catroid/res/layout/dialog_text_multiline_dialog.xml]: 0dp? Is this line necessary?

**Comment by thmq** [comment]
please squash the commit correctly and consider @thomasmauerhofer 's suggestions.

**Pull request closed without merging**